test_wasm_builtin_simd is failing with latest llvm version

```
test_wasm_builtin_simd (test_core.wasm0) ... /usr/local/google/home/sbc/dev/wasm/emscripten/tests/test_wasm_builtin_simd.cpp:73:10: error: use of undeclared identifier '__builtin_wasm_swizzle_v8x16'; did you mean '__builtin_wasm_swizzle_i8x16'?
  return __builtin_wasm_swizzle_v8x16(x, y);
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
         __builtin_wasm_swizzle_i8x16

```

--- a/tests/test_core.py
+++ b/tests/test_core.py
@@ -5852,7 +5852,6 @@ return malloc(size);
 
   @wasm_simd
   @is_slow_test
-  @disabled("https://github.com/emscripten-core/emscripten/issues/13991")
   def test_wasm_builtin_simd(self):
     # Improves test readability
     self.emcc_args += ['-Wno-c++11-narrowing', '-Wno-format']

--- a/tests/test_wasm_builtin_simd.cpp
+++ b/tests/test_wasm_builtin_simd.cpp
@@ -70,7 +70,7 @@ i32x4 TESTFN i32x4_shuffle_reverse(i32x4 vec) {
   return __builtin_shufflevector(vec, vec, 3, 2, 1, 0);
 }
 i8x16 TESTFN v8x16_swizzle(i8x16 x, i8x16 y) {
-  return __builtin_wasm_swizzle_v8x16(x, y);
+  return __builtin_wasm_swizzle_i8x16(x, y);
 }
 i8x16 TESTFN i8x16_splat(int32_t x) {
   return (i8x16) {x, x, x, x, x, x, x, x, x, x, x, x, x, x, x, x};

